lib/erb_lint/linters/deprecated_classes.rb

```diff
@@ -154,7 +151,9 @@ def attributes(attributes_string)
 
             # The 3 captures [3..5] are the possibilities specified in ATTRIBUTE_VALUE_PATTERN
             possible_value_formats = attribute_matching_group[3..5]
-            value = possible_value_formats.reduce { |a, e| a.nil? ? e : a }
+            value = possible_value_formats.reduce do |first_val, second_val|
+              first_val.nil? ? second_val : first_val
```

Comment:
@lemonmade lets continue the discussion on this line here.

The reason it's done like this is because of the [pattern](https://github.com/Shopify/erb-lint/blob/4c550f93643cd9f80896e02b4c2053c2ae900f35/lib/erb_lint/linters/deprecated_classes.rb#L73-L77) which allows for 3 possible formats for the attribute value. After performing the regex match, all three of the capturing groups remain but only one of them actually successfully captured the value.

Do you have any ideas of how to work around this?